ethcore/types/src/client_types.rs

```diff
@@ -62,6 +62,27 @@ pub struct ClientReport {
 	pub gas_processed: U256,
 	/// Memory used by state DB
 	pub state_db_mem: usize,
+	/// I/O statistics for state DB.
+	pub io_stats: IoStats,
+}
+
+/// I/O statistics.
+#[derive(Default, Debug, Clone, Eq, PartialEq)]
+pub struct IoStats {
```

Comment:
This type, seems to be the same as [kvdb::IoStats](https://github.com/paritytech/parity-common/blob/master/kvdb/src/io_stats.rs#L21-#L40) without time? 

Why not use `kvdb::IoStats`? It is because it is not `Default` or what?

Reply:
To avoid dependency on `kvdb` and decouple types, such that we can only include fields we're interested in.